tools/cmake/liblet.cmake

```diff
@@ -45,6 +45,13 @@ if(CMAKE_CXX_COMPILER_ID MATCHES "Clang")
   set(CMAKE_CXX_ARCHIVE_FINISH "<CMAKE_RANLIB> -no_warning_for_no_symbols -c <TARGET>")
 endif()
 
+if(CMAKE_CXX_COMPILER_ID MATCHES "MSVC")
+  set(WIN_TARGET_ARCH $ENV{VSCMD_ARG_TGT_ARCH})
```

Comment:
@vmoroz In the other draft review, you were asking about this variable and how it was set. VSCMD_ARG_TGT_ARCH is set by the Visual Studio command shell (vcvarsall.bat and its wrappers). When it is missing, I fail with a message (ln 51) that asks the user if they are in a Visual Studio shell, to help them realize this is a requirement.

Reply:
Makes sense. Thank you!